github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-periodics.yaml

```
@@ -107,3 +107,45 @@ periodics:
       resources:
         requests:
           memory: "8Gi"
+- name: periodic-containerized-data-importer-push-nightly-s390x
+  cron: "2 3 * * *"
+  decorate: true
+  annotations:
+    testgrid-create-test-group: "false"
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  max_concurrency: 1
+  labels:
+    preset-dind-enabled: "true"
```

This preset won't work with the container image below. 
```suggestion
    preset-podman-in-container-enabled: "true"
```

---

@awels I can't remember why the CDI nightlies are using the old docker-in-docker setup ?

---

No particular reason, probably because we never updated it.